### ISSUE TITLE
a gene name to input gene conversion table

### ISSUE THREAD

**Issue by ChristopherMancuso** [krishnanlab/PyGenePlexus#316]
need to check these files exist in the backend data then should be easy to implement. may need to check if these extra files will make us need a larger function for this part

**Comment by vincerubinetti** [comment]
To clarify, the request here is to put include gene "names" in the `input_genes` field of the analysis results, e.g.:

```js
{
  input_genes: [
    {
      input: "CASP3",
      entrez: "836",
      in_network: true,

      // add
      name: "Caspase 3"
    },

    // ... another gene
  ]
}
```

**Comment by ChristopherMancuso** [comment]
yeah the is what I'm getting from Arjun's request too, but probably the order would be Input, Name, Entrez. I'm iffy on this as Names can make the data frame much wider and look not so great as you know, but we can give it a try.

**Comment by ChristopherMancuso** [comment]
fixed with #325
